src/PackagePrivate/Parser/Parser.php

```diff
@@ -259,18 +261,122 @@ public function buildSet(string $input): Set
 				$sets[] = (new Parser())->createEdtf($matches[1]);
 			}
 			elseif(false != preg_match('/(.+)\.\.(.+)/', $value, $matches)){
-				$start = (int)$matches[1];
-				$end = (int)$matches[2];
-				for($i=$start;$i<=$end;$i++){
-					$sets[] = (new Parser())->createEdtf((string)$i);
-				}
+                $fromExtDate = (new Parser())->createEdtf($matches[1]);
```

Comment:
You still haven't updated your IDE config for this project to use tabs instead of spaces? ;)